CollAction/CollAction.csproj

```diff
@@ -44,4 +40,7 @@ npm run build</PreBuildEvent>
   <ItemGroup>
     <Folder Include="wwwroot\js\" />
   </ItemGroup>
+  <ItemGroup>
+    <TypeScriptCompile Include="Frontend\app\global\RichTextEditor.tsx" />
```

Comment:
I thought we had our own frontend "builder" (through webpack, triggered through npm build) which takes care of the typescript. Had to explicitly disable the c# typescript stuff to stop it interfering. Why are we using the buildin typescript functionality here?